### PR TITLE
updated java handson lab content to latest sdk version

### PR DESCRIPTION
**Pull request by richagaur**

### CODE REVIEW

**Review by TheovanKraay — APPROVED** [review]
LGTM

**Comment by TheovanKraay** [comment]
You have not linked the main readme to the new change feed doc :-)

File: java/labs/02-load_data_with_adf.md [hunk 1]
```diff
@@ -48,85 +50,83 @@ You will use **Azure Data Factory (ADF)** to import the JSON array stored in the
...
+1. In the **Git Configuration** tab, check **Configure Git later** option (this may be unchecked by default). Click **Review + Create**.
+
+    ![Git options ADF](../media/03-adf_git_conf.jpg)
+
+1. After creation, open your newly created Data Factory. Select **Open Azure Data Factory Studio** and you will launch ADF. You should see a screen similar to the screenshot below. Select **Ingest Data**. We will be using ADF for a one-time copy of data from a source JSON file on Azure Blob Storage to a database in Cosmos DB’s SQL API. ADF can also be used for more frequent data transfers from Cosmos DB to other data stores.
+    ![ADF studio](../media/03-adf_studio.jpg)
+    ![Ingest data](../media/03-adf_ingest_data.jpg)
 
-1. Edit basic properties for this data copy. You should name the task **ImportNutrition** and select to **Run once now**. Do not select **enable git**.
+1. Edit basic properties for this data copy. Select **Built-in copy task** option and chose **Run once now**.
```
Comment (sajeetharan) [inline java/labs/02-load_data_with_adf.md:75]: typo 'choose'

**Review by markjbrown — APPROVED** [review]

File: java/solutions/pom.xml [hunk 2]
```diff
@@ -1,76 +1,133 @@
...
+  <dependencies>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-cosmos</artifactId>
+      <version>LATEST</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-slf4j-impl</artifactId>
+      <version>2.13.0</version>
```
Comment (markjbrown) [inline java/solutions/pom.xml:95]: Can you please bump this up to the latest version of log4j? I think it is at least 2.17.0

**Comment by markjbrown** [comment]
This is really great to see.

Overall looks great. Please update the main readme.md to include your new lab. Also, please update the log4j to the latest version in your solution.

Let me know when ready and I will merge.

Thanks!

File: java/solutions/src/main/java/com/azure/cosmos/handsonlabs/lab09/Lab09Main.java [hunk 3]
```diff
@@ -51,135 +50,154 @@ public class Lab09Main {
...
 
         List<Transaction> transactions = new ArrayList<Transaction>();
-        for (int i=0; i<100; i++) transactions.add(new Transaction());
+        for (int i = 0; i < 10000; i++)
+            transactions.add(new Transaction());
 
         /**
-         * Although this block of code uses Async API to insert Cosmos DB docs into a container,
-         * we are blocking on each createItem call, so this implementation is effectively Sync.
+         * Although this block of code uses Async API to insert Cosmos DB docs into a
```
[the remark this answers is not in the document]
Comment (richagaur) [reply java/solutions/src/main/java/com/azure/cosmos/handsonlabs/lab09/Lab09Main.java:93]: This is just informational text on how exactly to use Async api. Customer can try both implementations to see the difference.

**Review by sajeetharan — APPROVED** [review]
LGTM👍

**Pull request merged**